packages/tools/src/tools/base/ContourBaseTool.ts

```diff
@@ -219,6 +219,7 @@ abstract class ContourBaseTool extends AnnotationTool {
     },
     transforms: {
```

Comment:
This this be on base annotation tool?  Not sure that belongs in this PR, but seems like having the transforms available separately is a useful addition.